JavaSource/org/unitime/timetable/model/CourseOffering.java

```diff
@@ -306,6 +306,7 @@ public Object clone() {
         co.setExternalUniqueId(this.getExternalUniqueId());
         co.setScheduleBookNote(this.getScheduleBookNote());
         co.setIsControl(this.getIsControl());
+        co.setFundingDept(this.getFundingDept()); //TODO Get effective?
```

Comment:
This would not work -- it is needed to lookup the funding department in the new academic session, not link the one from the previous session. See how it is done for subject areas [here](https://github.com/UniTime/unitime/blob/dev_funding_dept/JavaSource/org/unitime/timetable/util/SessionRollForward.java#L993-L996).
```
if (fromSubjectArea.getFundingDept() != null) {
    Department toFundingDept = fromSubjectArea.getFundingDept().findSameDepartmentInSession(toSession);
    toSubjectArea.setFundingDept(toFundingDept);
}
```